Library/Formula/infer.rb

```diff
@@ -0,0 +1,60 @@
+class Infer < Formula
+  desc "Facebook Infer is a static analysis tool - if you give Infer some Objective-C, Java, or C code, it produces a list of potential bugs."
+  homepage "https://fbinfer.com/"
+  url "https://github.com/facebook/infer/releases/download/v0.1.0/infer-osx-v0.1.0.tar.xz"
```

Comment:
Thanks!

The [emily formula](https://github.com/mistydemeo/homebrew-emily/blob/master/emily.rb) might be a good reference - it sets up a private opamroot within the build directory and installs all its dependencies, including ocamlfind, there.